sycl/include/CL/sycl/info/info_desc.hpp

```diff
@@ -141,7 +141,9 @@ enum class device : cl_device_info {
   ext_intel_gpu_eu_count_per_subslice =
       PI_DEVICE_INFO_GPU_EU_COUNT_PER_SUBSLICE,
   ext_intel_max_mem_bandwidth = PI_DEVICE_INFO_MAX_MEM_BANDWIDTH,
-  ext_intel_mem_channel = PI_MEM_PROPERTIES_CHANNEL
+  ext_intel_mem_channel = PI_MEM_PROPERTIES_CHANNEL,
+  // currently only implemented for nvidia
+  atomic64 = PI_DEVICE_INFO_ATOMIC_64
```

Comment:
this would be an extension to SYCL, right? If such then we should add this similar to how we add other extensions: see https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/IntelGPU/IntelGPUDeviceInfo.md

Reply:
As I understand it this is not an extension to SYCL2020.  The atomic64 aspect is described in the SYCL2020 specification.  See for example page 83 of the latest specification.  Have I have misunderstood something?

Reply:
It's standard SYCL feature. See https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html.

According to my understanding this feature accumulates functionality of `cl_khr_int64_base_atomics` and `cl_khr_int64_extended_atomics` extensions.